rust/connlib/tunnel/src/client.rs

```diff
@@ -197,7 +199,7 @@ where
 
         self.callbacks.on_tunnel_ready()?;
 
-        tracing::debug!("background_loop_started");
+        tracing::debug!(ip4 = %config.ipv4, ip6 = %config.ipv6, "TUN device initialized");
```

Comment:
How about also printing the device name? We would need to expose an extra function for that but it will be helpful